src/Plugin.php

```diff
@@ -329,4 +353,19 @@ public function do_deactivation_process() {
 	public function register_scripts() {
 		wp_register_script( 'cc-woo-public', trailingslashit( plugin_dir_url( $this->get_plugin_file() ) ) . 'app/bundle.js', [ 'wp-util' ], self::PLUGIN_VERSION, false );
 	}
+
+	/**
+	 * Load back-end scripts.
+	 *
+	 * @author Biplav Subedi <biplav.subedi@webdevstudios.com>
+	 * @since  ??
```

Comment:
@bplv112 I am seeing `@since ??` on this whole PR. Can you please let me know why this is the case?

Adding a simple `@since 2.0.0` would have been fine or just even a date.

Reply:
Oh yeah i will replace the `??` with the current version number once the PRs are approved and it's ready to release on the main branch. It was what I used to do with PPU since it was there on their documentation. 
It is basically to avoid confusion and change the version number all together once it is finalized and cemented. 

I'll document this on confluence and also replace the `??` in this PR. 
@asharirfan